### ISSUE TITLE
westend: Error: findMetaCall: Unable to find Call with index 0x9004

### ISSUE THREAD

**Issue by islishude** [paritytech/substrate-api-sidecar#343]
**Description**

code: lastest commit
nodejs: v14 LTS
network: westend testnet

**Steps to Reproduce**

request for `/blocks/2223354` and get `Error: findMetaCall: Unable to find Call with index 0x9004`

**Expected vs. Actual Behavior**

should no error here

**Comment by emostov** [comment]
Thanks for the details on how to reproduce the issue.

**tl;dr** I have a temporary fix with a working branch here: https://github.com/paritytech/substrate-api-sidecar/tree/zeke-try-opaque

It looks like Sidecar is error-ing here: https://github.com/paritytech/substrate-api-sidecar/blob/eeea29b74ef50eb45356e4a7e1ea04344097cc00/src/services/blocks/BlocksService.ts#L451 
when it tries to decode the opaque call.

We have had issues here before, but they should have been fixed when we started using the registry specific to the block with this PR: https://github.com/paritytech/substrate-api-sidecar/pull/323/files. So I am not sure exactly what the issue is
